traits/src/food.rs

```diff
@@ -2,19 +2,15 @@ use crate::primitives::*;
 use codec::{Decode, Encode};
 use frame_support::pallet_prelude::*;
 use scale_info::TypeInfo;
-use sp_runtime::{DispatchError, DispatchResult, RuntimeDebug};
+use sp_runtime::RuntimeDebug;
+use sp_std::vec::Vec;
 
 /// Incubation Food info
-#[cfg_attr(feature = "std", derive(PartialEq, Eq))]
-#[derive(Encode, Decode, RuntimeDebug, TypeInfo, MaxEncodedLen)]
+//#[cfg_attr(feature = "std", derive(PartialEq, Eq))]
+#[derive(Encode, Decode, RuntimeDebug, TypeInfo)]
 pub struct FoodInfo {
 	/// Era that an account last fed food to another Origin of Shell.
 	pub era: EraId,
-	/// Food left to feed to another Origin of Shell or to self. Each account gets 5 food to feed
-	/// per day.
-	pub food_count: u8,
-	/// Food left to feed to self. Max of 2 to feed self per day.
-	pub food_count_for_self: u8,
-	/// Number of times fed in the current era.
-	pub food_received: u32,
+	/// A BoundedVec of (CollectionId, NftId)
+	pub origin_of_shells_fed: Vec<(CollectionId, NftId)>,
```

Comment:
Will make it a `BoundedVec` right?

Reply:
This Vec can only get to a size of 5 based on the logic. Does it need to be a `BoundedVec`?

Reply:
Yes, for completion